textile/chat-features.textile

```diff
@@ -402,7 +402,7 @@ Typing Indicators allow chat room users to indicate to others that they are typi
 ** @(CHA-T2d)@ @[Testable]@ If the room status is @Attached@, then the @get@ call shall invoke the underlying @presence.enter()@ call.
 ** @(CHA-T2e)@ @[Testable]@ If the room status is @Detached@, an @ErrorInfo@ with code 40000, explaining that attach must be called first, will be thrown.
 ** @(CHA-T2f)@ @[Testable]@ For any other room status, an @ErrorInfo@ with code 40000 will be thrown.
-* @(CHA-T3)@ @[Testable]@ Users may configure a timeout interval for when they are typing. This configuration is provided as part of the @RoomOptions@ @typing.timeoutMs@ property, or idiomatic equivalent. The default is 5000ms.
+* @(CHA-T3)@ @[Testable]@ Users may configure a timeout interval for when they are typing. This configuration is provided as part of the @RoomOptions@ @typing.timeoutMs@ property, or idiomatic equivalent. The default is 1000ms.
```

Comment:
@sacOO7 if we decide to go ahead with this change, please can you [mark this spec point as replaced](https://github.com/ably/specification/blob/main/CONTRIBUTING.md#additional-notes-on-features-spec-point-removal-and-replacement) since we've already implemented this spec point in Swift

Reply:
Sure, will do that.
Also, post your thoughts about default timeout / max capping for the same as per https://github.com/ably/specification/pull/238#issue-2690234910